daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -18,7 +18,7 @@ import org.slf4j.LoggerFactory
 
 import scala.collection.JavaConverters._
 import scala.util.control.NoStackTrace
-
+@SuppressWarnings(Array("org.wartremover.warts.ArrayEquals"))
```

Comment:
What is this about?

Reply:
I changed to the better idiom, as used in SExpr.scala `extends SomeArrayEquals`.